### PR TITLE
api: Send workflow item currency with subproject details

### LINKED ISSUE #281: ui: Currency of workflow items is not saved correctly
Steps to reproduce: 
* Create new workflow item, select amount type "allocated" or "disbursed" 
* Select currency != subproject currency
* Submit workflow item
* Displayed currency is again subproject currency instead of selected currency. This is OK since the value is calculated based on the given exchange rate. 
* When editing the workflow item, the pre-selected currency is again the currency of the subproject. Expected: Currency selected during creation/edit process.

### PR DESCRIPTION
**Pull request by mathiashoeld**
# Changes

When the subproject details are fetched, the workflow item currency is
now returned as well.
Removed the action 'fetchWorkflowItems' as it is not used.

Closes #281

### CODE REVIEW

File: frontend/src/pages/Workflows/WorkflowItem.js [hunk 1]
```diff
@@ -369,6 +370,7 @@ const renderActionButtons = (
           title={editDisabled ? "" : strings.common.edit}
           workflowSortEnabled={workflowSortEnabled}
           status={status}
+          dataTest="edit-workflowitem"
```
Comment (gonzochic) [inline frontend/src/pages/Workflows/WorkflowItem.js]: is it possible to pass "data-test"?

Comment (gonzochic) [reply frontend/src/pages/Workflows/WorkflowItem.js]:
```js
const foo = ({["bar-test"]: bar}) => {
  console.log(bar)
}

const props = {
  "bar-test": "yesss"
};

foo(props)
```

File: frontend/src/pages/Common/Identifier.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ class Identifier extends Component {
           helperText={this.props.nameHintText}
           value={this.props.name}
           onChange={this.props.nameOnChange}
-          id="nameinput"
+          dataTest="nameinput"
```
[the remark this answers is not in the document]
Comment (mathiashoeld) [reply frontend/src/pages/Common/Identifier.js]: done

File: frontend/src/pages/Common/Identifier.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ class Identifier extends Component {
           value={this.props.comment}
           onChange={this.props.commentOnChange}
           multiline={true}
-          id="commentinput"
+          dataTest="commentinput"
```
[the remark this answers is not in the document]
Comment (mathiashoeld) [reply frontend/src/pages/Common/Identifier.js]: done

File: frontend/src/pages/Workflows/WorkflowItem.js (hunk 1 above)
Comment (mathiashoeld) [reply frontend/src/pages/Workflows/WorkflowItem.js]: done

**Pull request merged**